llama-index-integrations/llms/llama-index-llms-qianfan/README.md

```diff
@@ -0,0 +1,22 @@
+# LlamaIndex Llms Integration: Baidu Qianfan
+
+Baidu Intelligent Cloud's Qianfan LLM Platform Chat Client: Implementing the CustomLLM Abstraction Class.
```

Comment:
we should add the install somewhere here: `pip install llama-index-llms-qianfan`